pkg/runtime/utils_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package runtime
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"syscall"
+	"testing"
+)
+
+const (
+	mockMetadata = `{
+  "version": "v1.19.8",
+  "arch": "amd64",
+  "ClusterRuntime": "kubeadm",
```

Comment:
it is easy to understand the test logic if we set value as `k0s` or `k8s`.

Reply:
Ok,  I agree this thought.